Makefile

```diff
@@ -156,9 +126,11 @@ install: build
 	cp -R bin/* $(HOME)/.porter/
 	ln -f -s $(HOME)/.porter/porter /usr/local/bin/porter
 
-clean:
+clean: clean-mixins clean-last-testrun
+
+clean-mixins:
 	-rm -fr bin/
-	-rm -fr cnab/
-	-rm porter.yaml Dockerfile bundle.json
-	-helm delete --purge porter-ci-mysql
-	-helm delete --purge porter-ci-wordpress
+
+clean-last-testrun:
+	-rm -fr cnab/ porter.yaml Dockerfile bundle.json
+	-helm delete --purge porter-ci-mysql porter-ci-wordpress
```

Comment:
This line can be removed now, right?  (No longer applicable now that helm releases have unique names and the corresponding test script(s) invoke a `porter uninstall`, etc.)